data.sql

```diff
@@ -7,4 +7,31 @@ VALUES
   ('Angemon', '2005-06-12', -45, true, 1),
   ('Boarmon', '2005-06-07', 20.4, true, 7),
   ('Blossom', '1998-10-13', 17, true, 3),
-  ('Ditto', '2022-05-14', 22, true, 4);
\ No newline at end of file
+  ('Ditto', '2022-05-14', 22, true, 4);
+
+  INSERT INTO owners(full_name, age)
+VALUES
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+
+INSERT into species(name)
+values
+  ('Pokemon'),
+  ('Digimon');
+
+update animals set species_id = case 
+  when name like '%mon' then (select id from species where name = 'Digimon')
+  else (select id from species where name = 'Pokemon')
+end;
+
+update animals set owner_id = case
+  when name = 'Agumon' then (select id from owners where full_name = 'Sam Smith')
+  when name = 'Pikachu' or name = 'Gabumon'  then (select id from owners where full_name = 'Jesnnifer Orwell')
+  when name = 'Plantmon' then (select id from owners where full_name = 'Bob')
```

Comment:
As per the project requirements, Bob owns two animals. *Plantmon* and *Devimon*. However, you have taken only one into account. Kindly fix this error. You will have to update the query results accordingly and update the screenshots where necessary. ⬇️
![image](https://github.com/lengushuru/Vet-Clinic-2/assets/33268927/374f9858-934e-4aa3-9554-03b84fc908c2)